login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx

```diff
@@ -9,12 +9,15 @@ import { useAuthContext } from '../../contexts';
  * Component renders a screen with contact information for support with the application.
  * Contact information is pulled from the context passed into the workflow.
  *
- * @param title to display in the header
- * @param icon to display in the icon
- * @param description to display the contact support content
+ * @param title the title for the screen
+ * @param icon the icon to display in the header
+ * @param emailSupportTitle to display the contact support title
+ * @param emailSupportContent to display the contact support content
+ * @param phoneSupportTitle to display the phone support title
+ * @param phoneSupportContent to display the phone support content
```

Comment:
Can we make all of these complete phrases? e.g.,
```suggestion
 * @param title the title for the screen
 * @param icon the icon to display in the header
 * @param emailSupportTitle text to display as the contact support title
 * @param emailSupportContent content to display as the contact support content
 * @param phoneSupportTitle text to display as the phone support title
 * @param phoneSupportContent content to display as the phone support content
```